examples/node-ava/test/lib/instance-of-test.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+const test = require('ava')
+
+class Person {
+  age () {}
+}
+
+class EightiesGuy extends Person {
+  hairspray () {
+    return true
+  }
+}
+
+test.beforeEach(function (t) {
+  t.context.td = require('../../../..')
```

Comment:
Is this relative path necessary? The `node/` example in the same repo is able to just do `global.td = require('testdouble')` by naming `"testdouble": "file:../.."` as a devDep in its package.json

Also, I don't think we should assign `td` in a beforeEach. It can happen once and should be safe at the top of a shared test helper and assigned globally.

Reply:
On my other machine I was not able to run `npm install` with the file dependency. It always errored with some error in the `.cache` folder. However on another machine it runs fine (maybe --no-package-lock) made the difference or a differen node/npm version. I'll check later.

Reply:
there are lots of issues pertaining to `file:` deps and package-lock. since this is just an example project, nuke the package lock if you need to and get it committed in a working state